acvm-repo/acir/Cargo.toml

```diff
@@ -18,26 +18,42 @@ workspace = true
 [dependencies]
 acir_field.workspace = true
 brillig.workspace = true
+noir_protobuf.workspace = true
+
+color-eyre.workspace = true
 serde.workspace = true
 thiserror.workspace = true
 flate2.workspace = true
 bincode.workspace = true
 base64.workspace = true
+prost.workspace = true
 serde-big-array = "0.5.1"
 strum = { workspace = true }
 strum_macros = { workspace = true }
 
+proptest = { workspace = true, optional = true }
+proptest-derive = { workspace = true, optional = true }
+
+[build-dependencies]
+prost-build.workspace = true
+protoc-prebuilt.workspace = true
+
 [dev-dependencies]
 serde_json = "1.0"
 serde-reflection = "0.3.6"
 serde-generate = "0.25.1"
 fxhash.workspace = true
 criterion.workspace = true
 pprof.workspace = true
+num-bigint.workspace = true
+
+acir = { path = ".", features = ["arb"] } # Self to turn on `arb`.
```

Comment:
Why are we having this crate depend on itself to turn on a feature flag? If we need to have this feature on all the time like this then we should just not have a feature flag.

Reply:
Ah, it's just for testing.

Reply:
It is for testing, yes, but I somehow again encountered the problem that if I just enabled this feature then `noir-wasm` didn't compile. I think it happened when it was like `acir = { workspace = true, features = ["arb"] }` instead of using `path`. 

Originally I tried to make it work like this:
```
cfg_attr(any(test, feature = "arb"), derive(proptest_derive::Arbitrary))]
```
The problem is that half of the stuff that needs `Arbitrary` live in the `brillig` crate, so for that to kick in I either need to add a `dev-dependency` on that crate with `arb`, which can trigger the `wasm` problem, or I can rely on the `arb` feature of `acir` turning on `brillig/arb`.